src/symbolize/gimli.rs

```diff
@@ -198,6 +199,7 @@ cfg_if::cfg_if! {
     } else if #[cfg(any(
         target_os = "macos",
         target_os = "ios",
+        target_os = "visionos",
         target_os = "tvos",
         target_os = "watchos",
     ))] {
```

Comment:
This has become a bunch of hot nonsense by now.
```suggestion
    } else if #[cfg(target_vendor = "apple")] {
```